definitions/checks/remote_execution/verify_settings_file_already_exists.rb

```diff
@@ -0,0 +1,42 @@
+module Checks::RemoteExecution
+  class VerifySettingsFileAlreadyExists < ForemanMaintain::Check
+    metadata do
+      description 'Check to verify remote_execution_ssh settings already exist'
+      tags :pre_upgrade
```

Comment:
I think we should add this manually to the 6.3 upgrade, as otherwise, this would apply also for `6.2.z` upgrades, where we don't want to do that.